package.json

```diff
@@ -483,6 +495,36 @@
 					"when": "view == PowerBIPipelines && viewItem =~ /.*,DEPLOY,.*/",
 					"group": "inline"
 				},
+				{
+					"command": "PowerBIPipelineStageArtifacts.deploy",
+					"when": "view == PowerBIPipelines && viewItem =~ /.*,PIPELINESTAGE,.*/ && viewItem =~ /.*,DEPLOYARTIFACTS,.*/",
+					"group": "inline"
+				},				
+				{
+					"command": "PowerBIPipelineStageArtifact.deploy",
+					"when": "view == PowerBIPipelines && viewItem =~ /.*,PIPELINESTAGEDASHBOARDS,.*/",
```

Comment:
instead of listing the different tpyes here, I would suggest to introduce a new itemType `PIPELINESTAGEARTIFACT` and set it in https://github.com/gbrueckl/PowerBI-VSCode/blob/main/src/vscode/treeviews/Pipelines/PowerBIPipelineStageArtifacts.ts#L66 accordingly instead of using the "parents" itemType which is wrong anyway